solutions/hw3/task_overview.md

```diff
@@ -0,0 +1,5 @@
+# Homework 3
+### Preparing data
+### Analyze
```

Comment:
Actually, the bottom place in **1c** (smallest fare) is tied: there are two people. You need to account for such cases.